owncloudApp/src/main/java/com/owncloud/android/providers/DocumentsStorageProvider.kt

```diff
@@ -313,10 +313,6 @@ class DocumentsStorageProvider : DocumentsProvider() {
 
     private fun createFolder(parentDocument: OCFile, displayName: String): String {
         val newPath = parentDocument.remotePath + displayName + OCFile.PATH_SEPARATOR
-        val serverWithForbiddenChars = isVersionWithForbiddenCharacters()
-        if (!FileUtils.isValidName(displayName, serverWithForbiddenChars)) {
```

Comment:
isValidName is not needed anymore, is it?

Reply:
Neither `isValidName` nor `isValidPath` of `FileUtils.jave` are needed so removing them

Reply:
Maybe we should keep checking if there are forbiddenCharacters like / * or \

Reply:
Sure, done